src/tests/test_api.py

```diff
@@ -0,0 +1,171 @@
+import unittest
+import xmlrpclib
+import base64
+import json
+from time import sleep
+
+from test_process import TestProcessProto
+
+
+class TestAPI(TestProcessProto):
+    _process_cmd = ['pybitmessage', '-t']
+    _threads_count = 13  # minus one ReceiveQueue
+    _seed = base64.encodestring(
+        'TIGER, tiger, burning bright. In the forests of the night'
+    )
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestAPI, cls).setUpClass()
+        cls.addresses = []
+        cls.api = xmlrpclib.ServerProxy(
```

Comment:
Check the URL better add it in the constant.

Reply:
For what purpose?

Reply:
In most cases I guess @omkar1117 has a point, but this is a unit test and I think it isn't necessary.